DigitalOffice/Tests/AuthenticationServiceTests/TokenTests/CreateNewTokenTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Moq;
+using NUnit.Framework;
+using LT.DigitalOffice.AuthenticationService.Token;
+using LT.DigitalOffice.AuthenticationService.Token.Interfaces;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.Text;
+namespace LT.DigitalOffice.AuthenticationServiceUnitTests.TokenTests
+{
+    public class CreateNewTokenTests
+    {
+        private Mock<IJwtSigningEncodingKey> signingEncodingKey;
+        private SymmetricSecurityKey expectedKey;
+        private NewToken newToken;
+
+        [OneTimeSetUp]
+        public void Initialization()
+        {
+            string securityKey = "qyfi0sjv1f3uiwkyflnwfvr7thpzxdxygt8t9xbhielymv20";
+
+            signingEncodingKey = new Mock<IJwtSigningEncodingKey>();
+
+            var tokenOptions = Options.Create(new TokenOptions
+            {
+                TokenAudience = "AuthClient",
+                TokenIssuer = "AuthClient",
+                TokenLifetime = 5
```

Comment:
5 (double)? Is it not a TimeSpan?
https://docs.microsoft.com/en-us/dotnet/api/system.identitymodel.tokens.sessionsecuritytokenhandler.tokenlifetime?view=netframework-4.8

Reply:
Is it not TimeSpane, this is confugurable in appsetting.json, I am rename this TokenLifetimeInMinutes.